src/components/idleMembers/card/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import classNames from '@/components/idleMembers/card/card.module.scss';
+
+type Props = {
+  idleMemberUserName: string
+}
+
+const Card: FC<Props> = ({ idleMemberUserName }) => {
+  const imageGenerator = (name: string) => `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/${name}/img.png`;
```

Comment:
I think we have this URL defined in the .env file, why not take it from there

ref - https://github.com/Real-Dev-Squad/website-status/blob/develop/.env